tests/unit/test_lexer.py

```diff
@@ -165,7 +165,7 @@ def testEmptyElementTagsDisabled(self):
 """
         ApacheConfigLexer = make_lexer(**options)
         tokens = ApacheConfigLexer().tokenize(text)
-        self.assertEqual(tokens, ['block/', '\n', 'block /', '\n', 'block hello/', '\n'])
+        self.assertEqual(tokens, [('block/',), '\n', ('block', ' ', '/'), '\n', ('block', ' ', 'hello/'), '\n'])
```

Comment:
Interesting - how a 109-char long line passed our flake8 job...?

Reply:
Lets merge these two PRs right away to make it quick and easy!

If you'll have a chance to get the CI failing on PEP8 issues - that would be great!